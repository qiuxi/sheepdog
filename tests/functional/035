#!/bin/bash

# Test sockfd connection

. ./common

trap "_uninit; exit \$status" 0 1 2 3 15
_uninit()
{
	for i in `seq 1 7`; do
		_cleanup_machine_simulation $i
	done
}

_need_to_be_root

for i in `seq 0 9`; do
	_start_sheep $i
done

_wait_for_sheep 10

_cluster_format -c 6

_vdi_create test 40M
_random | $DOG vdi write test &

sleep 2
# Test write timeout
for i in `seq 1 4`; do
	_simulate_machine_down $i
done

# wait for dog to finish
wait
_wait_for_sheep_recovery 0

$DOG vdi read test | md5sum > $STORE/csum.1

for i in `seq 0 9`; do
	$DOG vdi object -i $i test
done

for i in 10 11 12; do
	_start_sheep $i
done

_wait_for_sheep_recovery 0

# Test read timeout
$DOG vdi read test | md5sum > $STORE/csum.2 &
for i in `seq 5 7`; do
	_simulate_machine_down $i
done
wait

diff -u $STORE/csum.1 $STORE/csum.2
status=0
