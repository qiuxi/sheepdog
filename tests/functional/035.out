QA output created by 035
using backend plain store
Looking for the object 0x7c2b2500000000 (the inode vid 0x7c2b25 idx 0) with 6 nodes

127.0.0.1:7000 has the object (should be 6 copies)
127.0.0.1:7005 has the object (should be 6 copies)
127.0.0.1:7006 has the object (should be 6 copies)
127.0.0.1:7007 has the object (should be 6 copies)
127.0.0.1:7008 has the object (should be 6 copies)
127.0.0.1:7009 has the object (should be 6 copies)
Looking for the object 0x7c2b2500000001 (the inode vid 0x7c2b25 idx 1) with 6 nodes

127.0.0.1:7000 has the object (should be 6 copies)
127.0.0.1:7005 has the object (should be 6 copies)
127.0.0.1:7006 has the object (should be 6 copies)
127.0.0.1:7007 has the object (should be 6 copies)
127.0.0.1:7008 has the object (should be 6 copies)
127.0.0.1:7009 has the object (should be 6 copies)
Looking for the object 0x7c2b2500000002 (the inode vid 0x7c2b25 idx 2) with 6 nodes

127.0.0.1:7000 has the object (should be 6 copies)
127.0.0.1:7005 has the object (should be 6 copies)
127.0.0.1:7006 has the object (should be 6 copies)
127.0.0.1:7007 has the object (should be 6 copies)
127.0.0.1:7008 has the object (should be 6 copies)
127.0.0.1:7009 has the object (should be 6 copies)
Looking for the object 0x7c2b2500000003 (the inode vid 0x7c2b25 idx 3) with 6 nodes

127.0.0.1:7000 has the object (should be 6 copies)
127.0.0.1:7005 has the object (should be 6 copies)
127.0.0.1:7006 has the object (should be 6 copies)
127.0.0.1:7007 has the object (should be 6 copies)
127.0.0.1:7008 has the object (should be 6 copies)
127.0.0.1:7009 has the object (should be 6 copies)
Looking for the object 0x7c2b2500000004 (the inode vid 0x7c2b25 idx 4) with 6 nodes

127.0.0.1:7000 has the object (should be 6 copies)
127.0.0.1:7005 has the object (should be 6 copies)
127.0.0.1:7006 has the object (should be 6 copies)
127.0.0.1:7007 has the object (should be 6 copies)
127.0.0.1:7008 has the object (should be 6 copies)
127.0.0.1:7009 has the object (should be 6 copies)
Looking for the object 0x7c2b2500000005 (the inode vid 0x7c2b25 idx 5) with 6 nodes

127.0.0.1:7000 has the object (should be 6 copies)
127.0.0.1:7005 has the object (should be 6 copies)
127.0.0.1:7006 has the object (should be 6 copies)
127.0.0.1:7007 has the object (should be 6 copies)
127.0.0.1:7008 has the object (should be 6 copies)
127.0.0.1:7009 has the object (should be 6 copies)
Looking for the object 0x7c2b2500000006 (the inode vid 0x7c2b25 idx 6) with 6 nodes

127.0.0.1:7000 has the object (should be 6 copies)
127.0.0.1:7005 has the object (should be 6 copies)
127.0.0.1:7006 has the object (should be 6 copies)
127.0.0.1:7007 has the object (should be 6 copies)
127.0.0.1:7008 has the object (should be 6 copies)
127.0.0.1:7009 has the object (should be 6 copies)
Looking for the object 0x7c2b2500000007 (the inode vid 0x7c2b25 idx 7) with 6 nodes

127.0.0.1:7000 has the object (should be 6 copies)
127.0.0.1:7005 has the object (should be 6 copies)
127.0.0.1:7006 has the object (should be 6 copies)
127.0.0.1:7007 has the object (should be 6 copies)
127.0.0.1:7008 has the object (should be 6 copies)
127.0.0.1:7009 has the object (should be 6 copies)
Looking for the object 0x7c2b2500000008 (the inode vid 0x7c2b25 idx 8) with 6 nodes

127.0.0.1:7000 has the object (should be 6 copies)
127.0.0.1:7005 has the object (should be 6 copies)
127.0.0.1:7006 has the object (should be 6 copies)
127.0.0.1:7007 has the object (should be 6 copies)
127.0.0.1:7008 has the object (should be 6 copies)
127.0.0.1:7009 has the object (should be 6 copies)
Looking for the object 0x7c2b2500000009 (the inode vid 0x7c2b25 idx 9) with 6 nodes

127.0.0.1:7000 has the object (should be 6 copies)
127.0.0.1:7005 has the object (should be 6 copies)
127.0.0.1:7006 has the object (should be 6 copies)
127.0.0.1:7007 has the object (should be 6 copies)
127.0.0.1:7008 has the object (should be 6 copies)
127.0.0.1:7009 has the object (should be 6 copies)
